wavefront/server/apps/floconsole/floconsole/controllers/app_user_controller.py

```
@@ -0,0 +1,280 @@
+    # Revoke access
+    try:
+        await app_user_service.revoke_app_access(user_id, app_id)
+        logger.info(f'Revoked user {user_id} access to app {app_id}')
+
+        return JSONResponse(
+            status_code=status.HTTP_200_OK,
+            content=response_formatter.buildSuccessResponse(
+                {'message': 'App access revoked successfully'}
+            ),
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)